src/wiseagents/agents/collaboration_wise_agents.py

```diff
@@ -413,7 +413,7 @@ def process_request(self, request: WiseAgentMessage):
         """
         ctx = WiseAgentRegistry.get_or_create_context(request.context_name)
         chat_id = request.chat_id
-        if chat_id is not None:
+        if chat_id is not None and chat_id in ctx.llm_chat_completion != {}:
```

Comment:
Shouldn't this be `if chat_id is not None and chat_id in ctx.llm_chat_completion`?

Note that this code is removed in my PR and instead this is now part of WiseAgent#handle_response:

https://github.com/wise-agents/wise-agents/pull/292/files#diff-3490f048922c0bfc10a5552ce71d323d1d4d3a868e169cce91097d58922d9b5aR1033

Reply:
The correct one is IMHO if chat_id is not None and ctx.llm_chat_completion != {}: